packages/zoltan2/example/geometric/rcb_C.cpp

```diff
@@ -429,10 +426,6 @@ int main(int argc, char *argv[])
   delete problem3;
```

Comment:
Best practice is to use `unique_ptr` for objects that need to be managed by pointer, yet have local scope.

Reply:
Will discuss with team - there are a bunch of places like this so perhaps better done in a separate job since this is not new code. @kddevin

Reply:
Naked `new` and `delete` is an antipattern specifically called out in the C++ Core Guidelines.  For example:

  - [P.3](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#p3-express-intent)
  - [R.1](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rr-raii)
  - [R.11 ("Avoid calling new and delete explicitly")](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rr-newdelete)

I'm not your boss and perhaps you didn't write this code, so I won't tell you what to do here.  I'm just pointing out the above.